LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -618,9 +620,20 @@ - (void)_dismissMe {
 #pragma mark - UI setup
 - (void)_setupNavBarWithLogoutTitle:(NSString *)logoutTitle {
     // Navigation Bar with custom UI
-    self.navBar =
-    [[UINavigationBar alloc] initWithFrame:CGRectMake(0, LTHMainWindow.frame.origin.y,
-                                                      LTHMainWindow.frame.size.width, 64)];
+    if (LTHiOS11) {
+        UIView *patchView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, LTHMainWindow.frame.size.width, [LTHPasscodeViewController getStatusBarHeight])];
+        patchView.backgroundColor = [UIColor whiteColor];
+        [self.view addSubview:patchView];
+        self.navBar =
+        [[UINavigationBar alloc] initWithFrame:CGRectMake(0, [LTHPasscodeViewController getStatusBarHeight],
```

Comment:
While not wrong, we want to place the `navBar` "below the patch", so, semantically, a `y` of `patch.size.height` is a bit more correct.